.pre-commit-config.yaml

```
@@ -17,18 +17,18 @@ repos:
     exclude: .patch
 
 - repo: https://github.com/psf/black
```

For a faster mypyc-compiled wheel:
```suggestion
- repo: https://github.com/psf/black-pre-commit-mirror
```

---

Not for right now, but maybe at some point we should switch to ruff instead of black? Presumably it's even faster...

https://docs.astral.sh/ruff/formatter/

---

:+1: I use the ruff formatter on several of my projects and have been very satisfied with it.